### PR TITLE
feat: add lambda ruby wrapper for layer

### PR DESCRIPTION
**Pull request by xuan-cao-swi**
This PR continues the work from https://github.com/open-telemetry/opentelemetry-lambda/pull/918.
Since ruby lambda instrumentation is [released](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/721) ([rubygems](https://rubygems.org/gems/opentelemetry-instrumentation-aws_lambda)), so it's time to create the actual wrapper for it.

### CODE REVIEW

File: ruby/src/layer/wrapper.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'opentelemetry/sdk'
+require 'opentelemetry/exporter/otlp'
+require 'opentelemetry/instrumentation/aws_lambda'
+
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Instrumentation::AwsLambda'
+end
+
+::OpenTelemetry::Instrumentation.registry.register('OpenTelemetry::Instrumentation::AwsLambda')
+::OpenTelemetry::Instrumentation.registry.install(['OpenTelemetry::Instrumentation::AwsLambda'])
```
Comment (mwear) [inline ruby/src/layer/wrapper.rb]: I'm curious about these two lines. Doesn't `Instrumentation::Base` do this automatically?

Comment (xuan-cao-swi) [reply ruby/src/layer/wrapper.rb]: ah, yes, it should do it automatically, I will remove the registry part.

**Review by mwear — APPROVED** [review]
LGTM

**Comment by tylerbenson** [comment]
Thanks @mwear.

@xuan-cao-swi Before merging this and testing the release process, please update the `release-ruby-layer.yml` `release-group: prod` to `dev` (or remove that line). After we're happy with the build artifact, we can change it back to `prod`.

**Comment by xuan-cao-swi** [comment]
> @xuan-cao-swi Before merging this and testing the release process, please update the `release-ruby-layer.yml` `release-group: prod` to `dev` (or remove that line). After we're happy with the build artifact, we can change it back to `prod`.

Thanks @tylerbenson , I have updated to dev

**Review by tylerbenson — APPROVED** [review]

**Pull request merged**